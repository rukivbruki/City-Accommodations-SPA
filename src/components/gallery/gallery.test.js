import React from "react";
import renderer from "react-test-renderer";
import Gallery from "./gallery";

it(`Gallery component render correct`, () => {
  const photos = [
    `img/apartment-01.jpg`,
    `img/apartment-02.jpg`,
    `img/apartment-03.jpg`
  ];

  const gallery = renderer
    .create(<Gallery photos={photos}/>)
    .toJSON();

  expect(gallery).toMatchSnapshot();
});
