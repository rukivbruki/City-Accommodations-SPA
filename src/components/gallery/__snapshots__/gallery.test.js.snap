// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery component render correct 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/apartment-01.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/apartment-02.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/apartment-03.jpg"
      />
    </div>
  </div>
</div>
`;
